dist/tools/firmware_metadata/Makefile

```diff
@@ -0,0 +1,19 @@
+RIOTBASE := ../../..
```

Comment:
I think since dist/tools/ is a path that will always exists in any clone of RIOT, we don't really need to define `$(CURDIR)`, and so the relative path will always be the same.